### PR TITLE
Resources, About Us and News Page Components

### PR DESCRIPTION
**Pull request by puritybirir**
##### What does this PR do?
- It is a combination of work done on Resources, About Us and News Page Components

##### Description of Task to be completed?
 - Create The Resources Page
 - Create The Cards Component
 - Add get involved Component
 - Add Footer Component
 - Create the News Page Header
 - Create the About Us Header
 - Create the About Gill Component
 - Add the reusable components
 - Create the Our Partners Component

##### Any background context you want to provide?
- None

##### What are the relevant Pivotal Tracker stories?
#161791989
#161792013
#161791966

##### Questions?
- None

##### Screenshots?
- Resources Page
resourcespage

- About Us Page
aboutus

-  News Page
newspage

### CODE REVIEW

**Comment by kilemensi** [comment]
Hey @puritybirir. I think there is a bad/incomplete merge on this PR.

File: package.json [hunk 1]
```diff
@@ -5,10 +5,16 @@
...
+    "@fortawesome/fontawesome-svg-core": "^1.2.12",
+    "@fortawesome/free-brands-svg-icons": "^5.6.3",
+    "@fortawesome/free-regular-svg-icons": "^5.6.3",
+    "@fortawesome/free-solid-svg-icons": "^5.6.3",
+    "@fortawesome/react-fontawesome": "^0.1.4",
     "@material-ui/core": "^3.4.0",
     "@mdi/font": "^3.0.39",
     "@mdi/js": "^3.0.39",
     "@mdi/react": "^1.1.0",
+    "add": "^2.0.6",
```
Comment (kilemensi) [inline package.json]: Do we really need this package? And with Material UI and `@mdi` already included, do we also need `bootstrap-css-only` below?

File: package.json [hunk 2]
```diff
@@ -30,7 +36,8 @@
     "react-jss": "^8.6.1",
     "react-player": "^1.7.0",
     "react-wavesurfer": "^0.8.6",
-    "webfontloader": "^1.6.28"
+    "webfontloader": "^1.6.28",
+    "yarn": "^1.13.0"
```
Comment (kilemensi) [inline package.json]: Is `yarn` really a runtime dependency?

File: src/components/AboutImportantHeader/index.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    backgroundColor: '#ffffff57',
+    padding: '15px 41px',
+  },
+};
+
+function AboutImportantHeader({ classes }) {
+  return (
+    <div>
+      <div className={classes.parentContainer}>
+        <div>
```
Comment (kilemensi) [inline src/components/AboutImportantHeader/index.js]: What are these extra `<div>`s for?

File: src/components/AboutUsHeader/index.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    right: '1415px',
+    top: '145px',
+
+  },
+};
+
+function AboutUsHeader({ classes }) {
+  return (
+    <ResourceAboutHeader
+      className={classes}
```
Comment (kilemensi) [inline src/components/AboutUsHeader/index.js]: I don't think `className` is meant to be used like this... I know it works but I'd rather you create a different property like `classes` and use that to pass these classes.

File: src/components/IndexNav/index.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    marginRight: 'auto',
+  },
+
+};
+
+const ListLink = ({
+  listClass, to, customClass, linkText,
+}) => (
+  <li className={listClass}>
+    <Link to={to} className={customClass}>{linkText}</Link>
```
Comment (kilemensi) [inline src/components/IndexNav/index.js]: React supports `children` property that you could have used here
```javascript
const function ListLink({ listItemClassName, to, linkClassName, children }) {
  <li className={listItemClassName}>
    <Link to={to} className={linkClassName}>{children}</Link>
```

And the usage will be:
```javascript
<ListLink to="/" listClass={classes.firstChild}>{image}</ListLink>
```

File: src/components/ResourceCards/index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+class ResourceVideos extends Component {
+    state = {
+      resourceVideos: resourceCardContent,
+    }
+
+    render() {
+      const { resourceVideos } = this.state;
+      return (
+        <React.Fragment>
+          <CssBaseline />
```
Comment (kilemensi) [inline src/components/ResourceCards/index.js]: `CssBaseline` is already included in all pages using `withRoot`

**Review by kilemensi — CHANGES_REQUESTED** [review]
Hey @puritybirir. Solid progress so far.

Lets fix a few obvious UX & code issues and I think we'll be very close to an MVP.

UX
==

Please go through the design again and try to match it as much as you can. Some obvious stuff I could pick up by just looking:

 + Broken components,

screenshot from 2019-01-22 12-03-45

 + Misaligned carousel tabs,

screenshot from 2019-01-22 12-03-31

 + Needs more room between logo and hero.

screenshot from 2019-01-22 12-05-44

 + Too much space between the story cards or the story cards need to be wider,
 + All text content has the same width from the top of the page to the footer. If you look at the main headers in the design you'll see they start at the same left margin. In your implementation this vertical rhythm is lost, and
 + etc.

Code
====

 + Lets keep only two folder structure: `components` and `pages`. No need for reusable components folder, etc.
 + _Single_ file components do not need folders. It's fine to just use that file. Case in point, `More` should just be `components/More.js` and not `components/More/index.js`. The folder adds no value.
 + _Multiple_ file components *must* have their own folders. Case in point, `Blast`, it is correctly in `components/Blasts/index.js` since it's implementation also relies on `components/Blasts/blastSvg.js`
 + Page specific components may include the page name in their name _if/when_ there is a clash in names. So `AboutHeader` and `IndexHeader` are just two header components for the about and index pages respectively. But *all* components, page components or not, still must follow the above rules. For example in our code we have `IndexHeaderText` and `IndexHeaderCarousel`. If these components mean what they should, it means they're only for the index page header. This means they can all move into `IndexHeader` components as `components/IndexHeader/Text.js` and `components/IndexHeader/Carousel` with the main file being `components/IndexHeader/index.js`.
 + Use of headers like `h1` directly means we're not taking advantage of any typographical scaling/design provided by Material UI. We should really use material ui more (including theme support).

File: package.json (hunk 1 above)
Comment (puritybirir) [reply package.json]: Bootstrap css only is needed for the mdbreact carousel. After the MVP I think I will have to rebuild the whole carousel from scratch and stop using a library as it has proved to be extensively problematic

File: src/components/IndexNav/index.js (hunk 5 above)
Comment (puritybirir) [reply src/components/IndexNav/index.js]: Done

**Pull request merged**